src/goInstallTools.ts

```diff
@@ -33,6 +33,7 @@ const allTools: { [key: string]: string } = {
 	'gogetdoc': 'github.com/zmb3/gogetdoc',
 	'goimports': 'golang.org/x/tools/cmd/goimports',
 	'goreturns': 'github.com/sqs/goreturns',
+	'goformat': 'github.com/mbenkmann/goformat/goformat',
```

Comment:
@mbenkmann When I run `go get github.com/mbenkmann/goformat/goformat`, I get the error "package github.com/mbenkmann/goformat/goformat: cannot find package "github.com/mbenkmann/goformat/goformat" in any of:..."

Does it expect that the user has already cloned the repo?